src/services/swap_service.ts

```diff
@@ -52,9 +52,10 @@ export class SwapService {
         const swapQuoterOpts: Partial<SwapQuoterOpts> = {
             chainId: CHAIN_ID,
             expiryBufferMs: QUOTE_ORDER_EXPIRATION_BUFFER_MS,
-            liquidityProviderRegistryAddress: LIQUIDITY_POOL_REGISTRY_ADDRESS,
```

Comment:
This should not have been removed.

Reply:
Addressed in https://github.com/0xProject/0x-api/pull/162/commits/29d62150c006472a07c9a3e0f7bb65548b4a42a8